SECURITY.md

```diff
@@ -0,0 +1 @@
+see https://github.com/inyokaproject/inyoka/tree/staging/docs/security.rst
```

Comment:
To make this branch-independent.

```suggestion
See [SECURITY.rst](docs/security.rst).
```

Reply:
I doubt that will work on https://github.com/inyokaproject/inyoka/security

But we can try